crawl-ref/source/dat/descript/quotes.txt

```diff
@@ -1526,14 +1526,14 @@ fire dragon scales
 flail
 
 “Even after forcing their way, with great effort and loss, through this double
- defense, [the Germans] still found themselves at a disadvantage; for their
- armor scarce enabled them to contend on equal terms with the uncouth but
+ defence, [the Germans] still found themselves at a disadvantage; for their
+ armour scarce enabled them to contend on equal terms with the uncouth but
  formidable weapons of their adversaries. The Bohemians were armed with long
  iron flails, which they swung with prodigious force. They seldom failed to
  hit, and when they did so, the flail crashed through brazen helmet, skull and
  all.”
     -James A. Wylie, _The History of Protestantism_, vol. I, book 3, ch. 15
-     “Jon Huss and the Hussite Wars”. 1878.
+     “John Huss and the Hussite Wars”. 1878.
```

Comment:
good catch, the book's name in the reference appears to have been incorrect (seems independent of aus though)

Reply:
The name seemed as unlikely to me as the American spellings, to be honest. I _was_ impressed to find a scan of the book online to check against.